backend/ee/danswer/server/seeding.py

```diff
@@ -23,6 +30,8 @@ class SeedConfiguration(BaseModel):
     admin_user_emails: list[str] | None = None
     seeded_name: str | None = None
     seeded_logo_path: str | None = None
+    personas: list[PersonaSnapshot] | None = None
```

Comment:
NOTE: we may want to simplify this in the future (e.g. present a simpler interface specifically for seeding), but for now this is fine